src/murmur/Server.cpp

```diff
@@ -1671,7 +1671,7 @@ void Server::connectionClosed(QAbstractSocket::SocketError err, const QString &r
 		QCoreApplication::instance()->postEvent(this,
 												new ExecEvent(boost::bind(&Server::removeChannel, this, old->iId)));
 
-	if (static_cast< int >(u->uiSession) < iMaxUsers * 2)
+	if (static_cast< int >(u->uiSession) > 0 && static_cast< int >(u->uiSession) < iMaxUsers * 2)
```

Comment:
The cast is there just to prevent a signed/unsigned comparison warning, right?

If that's the case, I propose the following:

```suggestion
	if (u->uiSession > 0 && static_cast< int >(u->uiSession) < iMaxUsers * 2)
```

Reply:
I think you're right. Fixed it.